lib/release/notes/log.rb

```diff
@@ -114,6 +116,14 @@ def log_grouped_commits(log:, title:) # rubocop:disable Metrics/AbcSize
         digest_title(title: title, log_message: "#{messages.join("\n")}\n")
       end
 
+      # @api private
+      def trim_commit_hashes(commit_hashes)
```

Comment:
Wasn't really sure what the best name for this method was but this made sense to me. Open to suggestions.